--- LASzip-3.4.4/src/CMakeLists.txt.orig	2024-04-17 08:32:41.000000000 +0200
+++ LASzip-3.4.4/src/CMakeLists.txt	2024-04-21 12:48:08.925703020 +0200
@@ -25,6 +25,8 @@ set(LASZIP_SOURCES
     laswritepoint.cpp
     laszip.cpp
     laszip_dll.cpp
+    laszipper.cpp
+    lasunzipper.cpp
     mydefs.cpp
 	lasmessage.cpp
 )
@@ -36,3 +38,4 @@ if(HAVE_UNORDERED_MAP)
     add_definitions(-DHAVE_UNORDERED_MAP=1)
 endif(HAVE_UNORDERED_MAP)
 LASZIP_ADD_LIBRARY(${LASZIP_BASE_LIB_NAME} ${LASZIP_SOURCES})
+LASZIP_ADD_INCLUDES("laszip" laszip.hpp laszipper.hpp lasunzipper.hpp)
