--- LASzip-3.4.4/src/CMakeLists.txt~	2024-12-15 22:13:25.000000000 +0100
+++ LASzip-3.4.4/src/CMakeLists.txt	2024-12-15 22:16:12.465325009 +0100
@@ -38,4 +38,4 @@
     add_definitions(-DHAVE_UNORDERED_MAP=1)
 endif(HAVE_UNORDERED_MAP)
 LASZIP_ADD_LIBRARY(${LASZIP_BASE_LIB_NAME} ${LASZIP_SOURCES})
-LASZIP_ADD_INCLUDES("laszip" laszip.hpp laszipper.hpp lasunzipper.hpp)
+LASZIP_ADD_INCLUDES("laszip" laszip.hpp laszipper.hpp lasunzipper.hpp mydefs.hpp)
